aten/src/ATen/native/mkldnn/Normalization.cpp

```diff
@@ -32,10 +32,57 @@ std::tuple<Tensor, Tensor, Tensor> mkldnn_batch_norm_backward(
 
 #include <ATen/native/mkldnn/MKLDNNCommon.h>
 #include <ATen/native/mkldnn/Utils.h>
+#include <ATen/native/layer_norm.h>
 
 namespace at {
 namespace native {
 
+std::tuple<Tensor, Tensor, Tensor> mkldnn_layer_norm(
+    const Tensor& input,
+    IntArrayRef normalized_shape, const c10::optional<Tensor>& weight_opt /* optional */, const c10::optional<Tensor>& bias_opt /* optional */,
+    double eps) {
+
+  TORCH_CHECK(weight_opt.has_value(), "optional weight isn't supported");
+  TORCH_CHECK(bias_opt.has_value(), "optional bias isn't supported");
+  auto weight = *weight_opt;
+  auto bias = *bias_opt;
+  TORCH_CHECK(normalized_shape.size() == 1, "only accept shapes with one dimension");
+  TORCH_CHECK(normalized_shape[0] == input.sizes().back(), "the shape dimension should be equal to the last dimension of input");
```

Comment:
wont this condition already be checked by `at::native::_check_layer_norm_inputs`? The quantization code does not include it https://github.com/pytorch/pytorch/blob/a9b0a921d592b328e7e80a436ef065dadda5f01b/aten/src/ATen/native/quantized/cpu/qnormalization.cpp#L26

Reply:
yeah, it does, I wanted a more specific error message though